test/server/specification.test.js

```diff
@@ -192,6 +180,94 @@ describe('Specification', () => {
           }
         })
       })
+
+      it(`[action: '${action}'] should set status to 'success' if title's ` +
+        `length is less than 'title.minimum-length.length' and ` +
+        `'title.minimum-length.enabled' is false`, async (done) => {
+        try {
+          const payload = createPayload(action, {
+            state: 'open',
+            title: 'ugh?',
+            body: 'This is a good body for the PR'
+          })
+
+          await pullRequest.execute(config({
+            title: {
+              'minimum-length': {
+                enabled: false
+              }
+            }
+          }), payload, TOKEN)
+          expect(github.setCommitStatus.calledWithExactly(
+            'sample', 'one', '1a2b3c', {
+              state: 'success',
+              context: 'zappr/pr/specification',
+              description: 'PR has passed specification checks'
+            }, 'token'
+          )).to.be.true
+          done()
+        } catch (e) {
+          done(e)
+        }
+      })
+
+      it(`[action: '${action}'] should set status to 'failure' if body contains ` +
```

Comment:
jfyi: In ES6 the backticks support multiline strings, so no need for  "` + \n" :)